frontend/src/components/EditRecipe/EditRecipeForm.js

```diff
@@ -76,6 +98,14 @@ const EditRecipeForm = ({ history }) => {
           </select>
         </div>
       </div>
+      <div>
+        <EditDirections
+          directions={directions}
+          handleDirectionsAdd={handleDirectionsAdd}
+          handleDirectionsChange={handleDirectionsChange}
+          handleDirectionsRemove={handleDirectionsRemove}
```

Comment:
Same question as the ingredients; what do you think about passing in `setDirections` here instead and moving the `handle*` methods into the directions component?

Reply:
@vhoof I was having trouble/getting an error when I tried that approach. setDirections was not recognized as a function when called in the child (EditDirections) component. I'll try again today and see if I can figure it out.

Reply:
@vhoof Not sure what the error was that I was running into yesterday. The latest commit should address this & work properly.